torch/autograd/gradcheck.py

```diff
@@ -182,40 +182,25 @@ def compute(a, b):
 
 def compute_numerical_jacobian_cols(jvp_fn, delta, input_is_complex, grad_out) -> List[torch.Tensor]:
     # Computing the jacobian only works for pure real or pure imaginary delta
-    # for details on the algorithm used here, refer:
+    # For details on the algorithm used here, refer:
     # Section 3.5.3 https://arxiv.org/pdf/1701.00392.pdf
     # s = fn(z) where z = x for real valued input
     # and z = x + yj for complex valued input
     jacobians_cols: List[torch.Tensor] = []
     ds_dx_tup = jvp_fn(delta)
+    assert(not isinstance(grad_out, complex))
```

Comment:
All the grad_out logic should be removed here right? This function is only ever called with grad_out=1 IIUC